aws_config_stream/main_config_stream.yaml

```
@@ -0,0 +1,241 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog AWS Config Changes Stream
+Parameters:
+  DataDogAPIKey:
```

It should be a small `d`

```suggestion
  DatadogAPIKey:
```